scripts/check_all_metrics.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euox pipefail
+
+all_metrics="backout component defectenhancementtask duplicate regression regressionrange regressor stepstoreproduce tracking"
```

Comment:
We should try to avoid hardcoding the models, maybe we could set the training tasks as dependencies of the "check_metrics" task, and the "check_metrics" task can get the metrics from its dependencies.

Reply:
As long as the naming scheme for the training tasks doesn't change, it should holds. I will work on that